### ISSUE TITLE
Device of messages not working

### ISSUE THREAD

**Issue by tik3ii** [tomer8007/whatsapp-web-incognito#184]
As the title says, it doesn't show device of messages anymore.

**Comment by tomer8007** [comment]
Yeah, I actually noticed that myself recently. It's probably because message decryption fails now. I will try to see.

**Comment by tomer8007** [comment]
Actually message decryption works and I can see the device of messages. Perhaps it's not working only for certain cases.

**Comment by tomer8007** [comment]
@tik3ii I believe this should be fixed now in version 2.2.0 (https://github.com/tomer8007/whatsapp-web-incognito/commit/1ca5692660a0d6b17a402669d7cceafbf0dd34c8). Please let me know if it works for you.

**Comment by tik3ii** [comment]
Device of message is working again!
